config/coreboot-galp5.config

```diff
@@ -758,7 +757,8 @@ CONFIG_TTYS0_LCS=3
 # CONFIG_SPKMODEM is not set
 # CONFIG_CONSOLE_NE2K is not set
 CONFIG_CONSOLE_CBMEM=y
-# CONFIG_CONSOLE_SPI_FLASH is not set
+CONFIG_CONSOLE_SPI_FLASH=y
+CONFIG_CONSOLE_SPI_FLASH_BUFFER_SIZE=0x20000
```

Comment:
@loftlifter31 this is console output (equivalent of cbmem console output) saved in SPI.
0x20000 might not be enough, and more debugging activation might be needed.

But this will permit to take a falshrom/flashprog backup externally and investigate the content in case of a bricked system and post those logs to make this go faster in case EFIFB/libgfxinit/GOP not working as expected.